src/components/TableCreator/tableCreator.tsx

```diff
@@ -63,13 +57,12 @@ const createRowSelectionCell = <T extends RowData>() =>
 
 export const TableCreator = <T extends RowData>({
   columns,
-  defaultSort,
   getRowId,
   items,
   listView,
   loading,
-  tableOptions,
 }: TableCreatorProps<T>): JSX.Element => {
+  const tableOptions = useTableOptions<T>();
```

Comment:
Table options are defined in the `TableCreator` component and passed through to the `Table` component!